File: website/Moe.La.Web/els-app/src/app/core/models/attachment.ts

import { Guid } from 'guid-typescript';

export class Attachment {
  id?: Guid;
  name: string = '';
  size?: number;
  file?: any;
  attachmentTypeId?: number;
  attachmentType?: string;
  isPhoto?: boolean;
  createdOn?: Date;
  createdOnHigri?: string = '';
  formateSize: string = '';
  isDraft= false;
  isDeleted= false;

}

export class HearingAttachment {
  id?: string;
  name: string = '';
  size?: number;
  file: any;
  attachmentTypeId?: number;
  isPhoto?: boolean;
  createdOnHigri?: string = '';
}

export class HearingUpdateAttachment {
  id?: string;
  name: string = '';
  size?: number;
  file?: any;
  attachmentTypeId?: number;
  isPhoto?: boolean;
}

export enum GroupNames {
  Case = 1,
  Hearing = 2,
  Memo = 3,
  CaseRule = 4,
  HearingUpdate = 5,
  RepresentativeLetterImage = 6,
  InvestigationRecord = 7,
  Moamala = 8
}

File: services/src/Moe.La.Core/Entities/Attachments/Attachment.cs
using System;

namespace Moe.La.Core.Entities
{
    public class Attachment : BaseEntity<Guid>
    {
        /// <summary>
        /// File name which came from user.
        /// </summary>
        public string Name { get; set; }

        /// <summary>
        /// File size
        /// </summary>
        public int? Size { get; set; }

        public bool IsDraft { get; set; } = true;

        public int? AttachmentTypeId { get; set; }

        public AttachmentType AttachmentType { get; set; }

        public Guid CreatedBy { get; set; }
    }
}
